wordpress-mobile/WordPress-iOS
# Zendesk Mobile: add support for source tags

Fixes #9290 

**NOTE:** 
Zendesk currently does not work if you are not logged in. If you want to test login/signup tags, in `ZendeskUtils:createIdentity`, change the `blog` guard to this:
```
guard let blog = blogService.lastUsedBlog() else {
    DDLogInfo("No Blog to create Zendesk identity with.")
    ZendeskUtils.sharedInstance.userEmail = "anyemailaddress@gmail.com"
    ZendeskUtils.createZendeskIdentity()
    completion(true)
    return
}
```

**To test:**
- In the app, access Support from a location that has a source tag. The list can be found in `WordPressSupportSourceTag+Helpers.swift`. 
  - The easiest/quickest may be the Site Creation flow > navigation bar 'Help'. 
- Select 'Contact Us' from the Support options.

To verify the origin tag is added to the ticket, you can either:
- Go ahead and submit a ticket.
- Find your ticket in Zendesk and verify the origin tag is present. (Feel free to ask me to look at your ticket. Might be easier.)

**OR**

- In `ZendeskUtils:getTags` log out the `tags` array before the method returns.
- When you select 'Contact Us' `tags` will appear in the logs.
- Verify the origin tag is present. 

@jleandroperez - can I bother you for a review please? Thank you!!
